<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Получить данные</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Открыть файл</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="variables">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>511</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="complexStructs">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>460</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="functions">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>511</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="chagePosition">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>0</y>
      <width>241</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="branches">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>460</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="mistakes">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>460</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
